src/NodaTime/LocalDate.cs

```diff
@@ -35,6 +36,7 @@ namespace NodaTime
 #if !NETSTANDARD1_3
     [Serializable]
 #endif
+    [DebuggerDisplay("{Year}-{Month}-{Day} {Calendar.Name}")]
```

Comment:
Let's *maybe* do this in a separate PR, but please remove it from this one.

Reply:
Sure. It was very confusing seeing the Wondrous dates being displayed as if they were Gregorian dates!